package.json

```
@@ -1,13 +1,14 @@
 {
   "name": "novuhq",
   "private": true,
+  "packageManager": "pnpm@7.5.0",
   "scripts": {
     "bootstrap": "npm run setup:dev",
     "start": "npm run jarvis",
     "preinstall": "npx only-allow pnpm",
     "prepare": "husky install",
     "publish": "lerna publish from-package",
-    "setup:project": "npx pnpm@7.5.0 i && node scripts/setup-env-files.js && npx pnpm build",
+    "setup:project": "pnpm i && node scripts/setup-env-files.js && pnpm build",
```

```suggestion
    "setup:project": "npx pnpm i && node scripts/setup-env-files.js && npm run build",
```

Wanted to have the initial setup of the project work without pnpm installed on your machine, and you would need it only if you want to add some dependencies. Wdyt, of only leaving the npx here?

---

Yeah we can do that, true 🙂

---

good point about the lock part. Agree with this.